### PR TITLE
Add Symfony 5 support

### PR DESCRIPTION
**Pull request by lucasmezencio**

### CODE REVIEW

**Comment by lucasmezencio** [comment]
Travis failed because of PHP 5.5 and 5.6. Should we drop it from Symfony 4+?

They already reached their EOL (even 7.1 reached it early this year), so I think it's feasible to do.

**Comment by maximecolin** [comment]
Hi @lucasmezencio. We can totally drop PHP 5.x and even 7.0. See our travis file from another bundle I recently updated for Symfony 5 : https://github.com/Elao/ElaoFormTranslationBundle/pull/34/files#diff-354f30a63fb0907d4ad57269548329e3

File: composer.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/framework-bundle": "~3.0|~4.0",
-        "symfony/form": "~3.0|~4.0"
+        "symfony/framework-bundle": "~3.0|~4.0|~5.1",
```
Comment (ogizanagi) [inline composer.json]: Why not allowing 5.0 ? (well, end of support is in one month, so why not)

Comment (lucasmezencio) [reply composer.json]: I can do it, no problem :)

**Comment by lucasmezencio** [comment]
All done, guys. Thanks. 🙂

**Review by Tom32i — APPROVED** [review]

**Comment by lucasmezencio** [comment]
Hey guys! Any news on this? 🙂

**Review by benji07 — APPROVED** [review]

**Comment by benji07** [comment]
Thank you @lucasmezencio

**Pull request merged**